.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
Yep. It's better for future compatibility, when 7.0 will be released, no need to change this.
Also it's more clear and doesn't change compared to `nightly`.